### ISSUE TITLE
`make test` fails in containerized build, 'overlay' not supported as root filesystem

### ISSUE THREAD

**Issue by stefan0xC** [bit-team/backintime#1184]
I'm running the build in a podman container but similar to #981 the test fails:

```
python -m unittest  -b test/test_tools.py
WARNING: import keyring failed
.....s..F.Fs.s..s......s................s..s.s...ss.s.s.............
======================================================================
FAIL: test_device (test.test_tools.TestTools)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/user/.cache/aurutils/sync/backintime/src/backintime-1.3.1/common/test/test_tools.py", line 446, in test_device
    self.assertRegex(tools.device('/nonExistingFolder/foo/bar'),
AssertionError: Regex didn't match: '(:?/dev/.*|tmpfs)' not found in 'overlay'

======================================================================
FAIL: test_filesystem (test.test_tools.TestTools)  
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/user/.cache/aurutils/sync/backintime/src/backintime-1.3.1/common/test/test_tools.py", line 452, in test_filesystem 
    self.assertRegex(tools.filesystem('/nonExistingFolder/foo/bar').lower(),
AssertionError: Regex didn't match: '(:?ext[2-4]|xfs|zfs|jfs|raiserfs|btrfs|tmpfs)' not found in 'overlay'

----------------------------------------------------------------------
Ran 68 tests in 0.486s

FAILED (failures=2, skipped=12)
make: *** [Makefile:577: unittest] Error 1
```

**Comment by yochananmarqos** [comment]
I can't reproduce that in a chroot, but the same test fails:

```
======================================================================
FAIL: test_powerStatusAvailable (test.test_tools.TestTools)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/backintime/src/backintime-1.3.1/common/test/test_tools.py", line 328, in test_powerStatusAvailable
    self.assertTrue(tools.powerStatusAvailable())
AssertionError: False is not true

----------------------------------------------------------------------
Ran 68 tests in 0.259s

FAILED (failures=1, skipped=9)
make: *** [Makefile:593: unittest-v] Error 1
make: Leaving directory '/build/backintime/src/backintime-1.3.1/common'
```

**Comment by buhtz** [comment]
Seems to be a duplicate or neighbour of #1389 which is fixed by 3a56a4c .

Can you confirm this with your podman setup?

**Comment by stefan0xC** [comment]
I think you're right. It seems to work just fine. Thanks for fixing the issue.
